python/py-awscli2/files/patch-requirements.diff

```diff
@@ -0,0 +1,31 @@
+diff -ru ./setup.cfg ../aws-cli-2.4.6/setup.cfg
+--- ./setup.cfg	2021-12-16 17:53:05.326741646 -0500
++++ ../aws-cli-2.4.6/setup.cfg	2021-12-16 17:56:05.149864718 -0500
+@@ -28,17 +28,17 @@
+ python_requires = >=3.7
+ include_package_data = True
+ install_requires =
+-    colorama>=0.2.5,<0.4.4
+-    docutils>=0.10,<0.16
+-    cryptography>=3.3.2,<3.4.0
+-    ruamel.yaml>=0.15.0,<0.16.0
+-    wcwidth<0.2.0
+-    prompt-toolkit>=2.0.0,<3.0.0
+-    distro>=1.5.0,<1.6.0
++    colorama
++    docutils
++    cryptography
++    ruamel.yaml
++    wcwidth
++    prompt-toolkit
++    distro
+     awscrt==0.12.4
```

Comment:
why do you leave the pinning here (the port you added is version 0.12.5)?

Reply:
I meant to package 0.12.4 and will fix that. This awscrt package seems pretty closely tied to awscli (as in I don't expect many other packages, if any, to ever use this dependency). At the same time awscli has its own release schedule, and they're up to 0.12.6. However, awscli is pinning the exact 0.12.4 version. Because of the close relationship of the two it seems prudent to keep the two versions in sync in MacPorts to minimize problems. So the pip machinery would continue relying on the exact bounds as specified by Amazon, and that would only change (and the MacPorts package for awscrt would only get a version update) when Amazon releases a new awscli with a requirement on the new awscrt.